### PR TITLE
C# XUnit 

### PR DESCRIPTION
**Pull request by abhi-singhs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
 # xunit-browserstack
+[XUnit](https://xunit.net/) Integration with BrowserStack.
+
+<img src="https://github.com/browserstack/cucumber-java-browserstack/blob/master/src/test/resources/img/browserstack.png?raw=true" width="60" height="60" alt="BrowserStack" > <img src="https://xunit.net/images/full-logo.svg" width="150" height="60" alt="XUnit">
+
+## Setup
+* Clone the repo
+* Install dependencies `dotnet build`
+* Update *.conf.json files inside the `XUnit-BrowserStack/conf` directory with your [BrowserStack Username and Access Key](https://www.browserstack.com/accounts/settings).
```
Comment (07souravkunda) [inline README.md]: Can we create a common config file at root for username and access key. Now user have to update in 3 files.

File: XUnit-BrowserStack/ParallelTest.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                driver.Navigate().GoToUrl("https://bstackdemo.com/");
+                Assert.Equal("StackDemo", driver.Title);
+
+                string productOnPageText = driver.FindElement(By.XPath("//*[@id=\"1\"]/p")).Text;
+
+                driver.FindElement(By.XPath("//*[@id=\"1\"]/div[4]")).Click();
+                bool cartOpened = driver.FindElement(By.XPath("//*[@class=\"float-cart__content\"]")).Displayed;
+                Assert.True(cartOpened);
+                string productOnCartText = driver.FindElement(By.XPath("//*[@id=\"__next\"]/div/div/div[2]/div[2]/div[2]/div/div[3]/p[1]")).Text;
+                Assert.Equal(productOnCartText, productOnPageText);
```
Comment (07souravkunda) [inline XUnit-BrowserStack/ParallelTest.cs:33]: Is timeout required to ensure tests don't fail?

**Review by 07souravkunda — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (abhi-singhs) [reply README.md]: I was trying to replicate what has been done in [this](https://github.com/browserstack/junit-browserstack) JUnit repo

File: XUnit-BrowserStack/ParallelTest.cs (hunk 2 above)
Comment (abhi-singhs) [reply XUnit-BrowserStack/ParallelTest.cs:33]: I will just add WebDriverWait for the find elements.

File: README.md (hunk 1 above)
Comment (07souravkunda) [reply README.md]: let's create a common config, this will reduce user steps

**Comment by 07souravkunda** [comment]
LGTM

**Review by 07souravkunda — APPROVED** [review]

**Pull request merged**